File: Clare Limo/ArraysAndStrings/reverseSentence.js
function reverseSentence(A){
    let str = A.join();
    let newA = str.split(' ');
    let n = newA.length;
    let res = "";

    for(let i=n-1;i>=0;i++){
        res+=newA[i]+" ";
    }
    

}

File: octacat/arrays-and-strings/reverse_sentence.py
def reverseSentence(A):
    word = ""
    for i in A: 
        word += i
    
    sentence_parts = list(word.split(' '))
    sentence_parts[0], sentence_parts[2] = sentence_parts[2], sentence_parts[0]
    sentence = ' '.join(sentence_parts)

    sentence = list(sentence)

    return(sentence)

A = ['t','h','i','s',' ','i','s',' ','g','o','o','d']
print(reverseSentence(A))